packages/common/src/schemas/index.ts

```diff
@@ -47,7 +48,7 @@ const trackMetadataSchema = {
 
 export const newTrackMetadata = (fields, validate = false) => {
   const validFields = validate
-    ? pick(fields, Object.keys(trackMetadataSchema).concat(['track_id', 'track_cid']))
+    ? pick(fields, Object.keys(trackMetadataSchema).concat(['track_id']))
```

Comment:
so do we need to concat track_id even since that does not matter?

Reply:
i have not double checked whether it matters and just wanted a quick fix track cid fix without changing what was there prior